README.md

```diff
@@ -1,41 +1,43 @@
 # Atleta Network
 
-The project is built on a [frontier template](https://github.com/PureStake/frontier) —
-[FRAME](https://docs.substrate.io/v3/runtime/frame)-based
-[Substrate](https://substrate.io) node with the Ethereum RPC support, ready for
+The project is built on a [frontier template](https://github.com/polkadot-evm/frontier) —
+[FRAME](https://docs.substrate.io/reference/)-based
+[Substrate](https://substrate.io) node with the [Ethereum RPC](https://ethereum.org/en/developers/docs/apis/json-rpc/#json-rpc-methods) support, ready for
 hacking :rocket:
 
 
 
 
 ## Build & Run
 
-#### Build with "fast-runtime" feature
-To build the test/dev network with the "fast-runtime" feature, execute the following command from the project root:
-```
-$ cargo build --release --features "fast-runtime" 
+#### Build _mainnet_ node
+To build the _mainnet_ network execute the following command from the project root:
+```shell
+cargo build --release
```

Comment:
actually, without any arguments, the `dev` chain type will be running.

Reply:
There are two weakly dependent flows: _build_ and _run_.
We can make build with or without features enabled. Either of these binaries can be run in `--dev` mode.
Here I mean default _noargs_ build, which is _mainnet_-compatible.
Maybe we should remove these _{dev,test,main}net_ mentions at all?

Reply:
And yeah, _no arguments_ node run isn't the same as `--dev`. It doesn't produce blocks.

<img width="1418" alt="image" src="https://github.com/user-attachments/assets/de43757c-abec-4b77-9024-95d90dc7e35a">

Reply:
—dev also specifies —alice and —validator. But the chainspec used when you don’t provide arguments is dev. Moreover, we didn’t specify chainspec for mainnet yet.

I agree, we can omit chain type mentioning.

Reply:
Mentions are removed.